changelog/upgrade-notes.md

```diff
@@ -2,6 +2,89 @@
 
 ### Next version
 
+#### [Breaking] Change of the symbol style format in `ol/layer/WebGLPointsLayer`
```

Comment:
I added a "Backwards incompatible changes" heading to [the upgrade notes](https://github.com/openlayers/openlayers/blob/fe8c2ebd5dfa9ba0e38af9bcc3352c0b617931b8/changelog/upgrade-notes.md).  Doesn't need to be in this change, but we should either group all the breaking changes under that header or add a prefix as you have done.

Reply:
rebased and moved this to the top of the upgrade notes